Kernel/Bus/VirtIO/Device.cpp

```diff
@@ -98,23 +100,23 @@ UNMAP_AFTER_INIT void Device::initialize()
             auto cfg = make<Configuration>();
             auto raw_config_type = capability.read8(0x3);
             if (raw_config_type < static_cast<u8>(ConfigurationType::Common) || raw_config_type > static_cast<u8>(ConfigurationType::PCI)) {
-                dbgln("{}: Unknown capability configuration type: {}", VirtIO::determine_device_class(address), raw_config_type);
+                dbgln("{}: Unknown capability configuration type: {}", m_class_name, raw_config_type);
```

Comment:
Technically, the `Device::initialize` method doesn't run in IRQ context, isn't it that correct?
Also, I'm not against this change (as it probably doesn't cost too much in memory allocation), but does it actually increase performance?

Reply:
I'm note sure if it increases performance, but if we have it stored already, what's the point of refetching it? (It most likely doesnt decrease performance, since this is just a single memory load, which is most likely at least as good as whatever GCC managed to optimize the function call into)